selfdrive/car/ford/radar_interface.py

```diff
@@ -8,27 +8,33 @@
 RADAR_MSGS = list(range(0x120, 0x12F))
```

Comment:
CADS does more than 16 points, seb and I nerfed it to 16 points in order to try and solve the amplitude issue, I believe it's a 64 point radar but cant remember off the top of my head,

Reply:
It does look like it yeah. In cabana I see a continuous range of messages from id 120 to 15F